skillmodels/tests/model_functions/transition_functions_test.py

```diff
@@ -36,29 +36,23 @@ def test_linear(setup_linear, expected_linear):
 
 
 # test for number of linear coefficients
-def test_nr_coeffs_linear():
-    included_factors = ["f1", "f2", "f3"]
-    assert (tf.nr_coeffs_linear(included_factors, "short")) == 3
+parameters = [(["f1", "f2", "f3"], "short", 3)]
```

Comment:
We also want two tests (one for params_type='short' and one for params_type='long') if the params type shouldn't have an effect.